data_processing/common/Node.py

```diff
@@ -82,6 +85,21 @@ def get_map_str(self):
 		return f"""{{ {prop_string} }}"""
 
 
+	def get_object(self, glob_string):
+		"""
+		Return a singular object for the data node given glob string
+		"""
+		return str(next(self.path.glob(glob_string)))
+	
+
+	def get_objects(self, glob_string):
+		"""
+		Return all objects for the data node given glob string
+		"""
+		if os.path.isdir(self.path):
+			return self.path.glob(glob_string)
+		else:
+			return self.path.parent.glob(glob_string)
```

Comment:
would it make sense for get_object() to call get_objects() and filter out a single object? took me a little bit to understand how these functions are being used. an example will be very helpful!

Reply:
Yeah... this is a bit of a mess rn, since sometimes the "path" is a file, but sometimes a folder with the files.  Might just need to iterate on this!